### ISSUE TITLE
MultiEPG EXIT action

### ISSUE THREAD

**Issue by MrQuade** [beyonwiz-dev/enigma2#15]
If the MultiEPG is invoked from the Event Info screen, the user can press EXIT, and is returned to Live TV.

If the Single EPG is invoked from the Event Info screen, the user can press EXIT, and is returned to the Event Info screen.

These two actions should behave consistently with each-other.

**Comment by atvcaptain** [comment]
yes i have see but no idea how can fix, when prl have time, please take a look
